man/journald.conf.xml

```diff
@@ -363,7 +363,38 @@
         <listitem><para>Takes a boolean value. If enabled (the
         default), journal reads <filename>/dev/kmsg</filename>
         messages generated by the kernel.</para></listitem>
-      </varlistentry>
+         </varlistentry>
+
+         <varlistentry>
+           <term><varname>FacilitiesStore=</varname></term>
+
+        <listitem><para>Controls the log facility of messages
+        that are stored on disk, forwarded to syslog, kmsg, the
+        console or wall (if that is enabled, see above). As argument,
+        takes one or more space separated facility names <option>kern</option>,
+        <option>user</option>, <option>mail</option>,
+        <option>daemon</option>, <option>auth</option>,
+        <option>syslog</option>, <option>lpr</option>,
+        <option>news</option>, <option>uucp</option>,
+        <option>cron</option>, <option>authpriv</option>,
+        <option>ftp</option>, <option>local0</option>,
+        <option>local1</option>, <option>local2</option>,
+        <option>local3</option>, <option>local4</option>,
+        <option>local5</option>, <option>local6</option> or
+        <option>local7</option>. See
+        <citerefentry project='man-pages'><refentrytitle>syslog</refentrytitle><manvolnum>3</manvolnum></citerefentry>
+        for details. This option is only useful when
+        <varname>StandardOutput=</varname> or
+        <varname>StandardError=</varname> are set to
+        <option>syslog</option>. Defaults to unset.</para>
+
+        <para>
+          Example:
+          <varname>FacilitiesStore=kern cron </varname> means only save LOG_KERN and LOG_CRON to disk,
+          drop messages from all others. Or, <literal>~</literal> syntax: <varname>FacilitiesStore=~kern daemon</varname>
+          implies store log messages of all facilities to disk, except for LOG_KERN and LOG_DAEMON</para>
```

Comment:
>  if some message is missing the SYSLOG_FACILITY then we fix it up by LOG_USER.

This happens when messages are forwarded to `syslog`, but I'm not sure how this is relevant to this PR.

> This is a rare case .

Could you clarify why you think so?

> Have you seen any daemon does this

Yes, I have. Those daemons could be found by searching for the code that uses `sd_journal_*` and doesn't pass `SYSLOG_FACILITY` explicitly.